GroovyDataCleanerExtension/pom.xml

```diff
@@ -36,9 +36,9 @@
 			<version>2.0.5</version>
 		</dependency>
 		<dependency>
-			<groupId>org.eobjects.analyzerbeans</groupId>
-			<artifactId>AnalyzerBeans-core</artifactId>
-			<version>0.22</version>
+			<groupId>org.eobjects.datacleaner</groupId>
+			<artifactId>DataCleaner-engine-core</artifactId>
+			<version>4.0-SNAPSHOT</version>
```

Comment:
I wanted to, but my Eclipse was stubborn, so only 4.0-RC2 existed according to it :)

I expect it to solve itself when releasing?